### PR TITLE
Move unused Navigation View Resources to Depricated_Resources.

### PR DESCRIPTION
**Pull request by StephenLPeters**
And cleanup some of the unneeded resource files.

### CODE REVIEW

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml [hunk 1]
```diff
@@ -169,50 +154,48 @@
         </ResourceDictionary>
 
         <ResourceDictionary x:Key="HighContrast">
-            <SolidColorBrush x:Key="NavigationViewDefaultPaneBackground" Color="{ThemeResource SystemColorWindowColor}" />
-            <SolidColorBrush x:Key="NavigationViewExpandedPaneBackground" Color="{ThemeResource SystemColorWindowColor}" />
-            <SolidColorBrush x:Key="NavigationViewTopPaneBackground" Color="{ThemeResource SystemColorWindowColor}" />
-            <SolidColorBrush x:Key="NavigationViewContentBackground" Color="{ThemeResource SystemColorWindowColor}" />
+            <StaticResource x:Key="NavigationViewDefaultPaneBackground" ResourceKey="AcrylicInAppFillColorDefaultBrush" />
```
Comment (karenbtlai) [inline dev/NavigationView/NavigationView_rs1_themeresources.xaml:157]: `NavigationViewDefaultPaneBackground` should be what the original was: 

<SolidColorBrush x:Key="NavigationViewDefaultPaneBackground" Color="{ThemeResource SystemColorWindowColor}" />

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml [hunk 2]
```diff
@@ -169,50 +154,48 @@
...
+            <StaticResource x:Key="NavigationViewTopPaneBackground" ResourceKey="AcrylicInAppFillColorDefaultBrush" />
```
Comment (karenbtlai) [inline dev/NavigationView/NavigationView_rs1_themeresources.xaml:158]: This as well: <SolidColorBrush x:Key="NavigationViewTopPaneBackground" Color="{ThemeResource SystemColorWindowColor}" />

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml [hunk 3]
```diff
@@ -109,32 +97,37 @@
             <StaticResource x:Key="NavigationViewItemBackgroundSelectedPressed" ResourceKey="SubtleFillColorSecondaryBrush" />
             <StaticResource x:Key="NavigationViewItemBackgroundSelectedDisabled" ResourceKey="SubtleFillColorSecondaryBrush" />
 
+            <StaticResource x:Key="NavigationViewItemBorderBrush" ResourceKey="SubtleFillColorTransparentBrush" />
+            <StaticResource x:Key="NavigationViewItemBorderBrushPointerOver" ResourceKey="SubtleFillColorTransparentBrush" />
+            <StaticResource x:Key="NavigationViewItemBorderBrushPressed" ResourceKey="SubtleFillColorTransparentBrush" />
+            <StaticResource x:Key="NavigationViewItemBorderBrushDisabled" ResourceKey="SubtleFillColorTransparentBrush" />
+            <StaticResource x:Key="NavigationViewItemBorderBrushChecked" ResourceKey="SubtleFillColorTransparentBrush" />
```
Comment (karenbtlai) [inline dev/NavigationView/NavigationView_rs1_themeresources.xaml:104]: Just did a quick search, it seems `NavigationViewItemBorderBrushChecked`, `NavigationViewItemBorderBrushSelected`, and its variants aren't used in the template as well. Should they be moved to the depracated themeresource?

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml (hunk 1 above)
Comment (StephenLPeters) [reply dev/NavigationView/NavigationView_rs1_themeresources.xaml:157]: This was overwritten to the value I have here in the NavigationView_rs2_themeresources.xaml file which I've deleted as a part of this change by merging those files into the rs1 file.

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml (hunk 2 above)
Comment (StephenLPeters) [reply dev/NavigationView/NavigationView_rs1_themeresources.xaml:158]: Same here.

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml (hunk 3 above)
Comment (StephenLPeters) [reply dev/NavigationView/NavigationView_rs1_themeresources.xaml:104]: thanks

**Comment by gabbybilka** [comment]
Thank you for submitting this PR, the current focus of the team is to assist our users in moving from WinUI 2 to WinUI 3. We are closing this PR as it does not address resolving a critical issue for WinUI 2 or WinUI 3.

**Pull request closed without merging**